rueidisaside/aside.go

```diff
@@ -210,6 +216,7 @@ func (c *Client) Close() {
 const PlaceholderPrefix = "rueidisid:"
 
 var (
-	delkey = rueidis.NewLuaScript(`if redis.call("GET",KEYS[1]) == ARGV[1] then return redis.call("DEL",KEYS[1]) else return 0 end`)
-	setkey = rueidis.NewLuaScript(`if redis.call("GET",KEYS[1]) == ARGV[1] then return redis.call("SET",KEYS[1],ARGV[2],"PX",ARGV[3]) else return 0 end`)
+	delkey    = rueidis.NewLuaScript(`if redis.call("GET",KEYS[1]) == ARGV[1] then return redis.call("DEL",KEYS[1]) else return 0 end`)
+	setkey    = rueidis.NewLuaScript(`if redis.call("GET",KEYS[1]) == ARGV[1] then return redis.call("SET",KEYS[1],ARGV[2],"PX",ARGV[3]) else return 0 end`)
+	setKeyLua = rueidis.NewLuaScript(`local set = redis.call("SET", KEYS[1], ARGV[1], "NX", "PX", ARGV[2]) if set then return nil else return redis.call("GET", KEYS[1]) end`)
```

Comment:
The new script looks good to me. However, could we make it even shorter? for example:
```suggestion
	setKeyLua = rueidis.NewLuaScript(`if redis.call("SET", KEYS[1], ARGV[1], "NX", "PX", ARGV[2]) then return nil else return redis.call("GET", KEYS[1]) end`)
```

Besides that, we also need tests that have the `UseLuaScripts` enabled in the `aside_test.go` file. You can copy all the existing tests and set the `UseLuaScripts` for them.